language/move-vm/runtime/src/loader.rs

```diff
@@ -1376,6 +1376,23 @@ impl<'a> Resolver<'a> {
         Type::Struct(struct_def)
     }
 
+    pub(crate) fn get_struct_ability(&self, idx: StructDefinitionIndex) -> AbilitySet {
```

Comment:
This is again very inefficient... Lets keep in mind to optimize

Reply:
This doesn't look very inefficient to me? Just reading out a value from a vector?